content/en/docs/features/get-resource.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Get Secret Resources 
+date: 2023-01-24
+description: Workloads that request resources from Trustee 
+categories:
+- feature 
+tags:
+- resources 
+---
+
+{{% alert title="Note" color="primary" %}}
+Requesting secret resources depends on attestation.
+[Configure attestation](../attestation) before requesting resources
+{{% /alert %}}
+
+While [sealed secrets](../sealed-secrets) can be used to connect unmodified workloads
+to resources from Trustee,
+a workload can also explicitly request resources via a REST API
+that is automatically exposed to the pod network namespace.
+
+For example, you can run this command from your container.
+```bash
+curl http://127.0.0.1:8006/cdh/resource/default/key/1
```

Comment:
Can You add additional information about how to create such resource i.e.:
https://github.com/confidential-containers/trustee/blob/main/kbs/docs/resource_storage_backend.md

and information about corresponding config section:

```toml
[[plugins]]
name = "resource"
type = "LocalFs"
dir_path = "/opt/confidential-containers/kbs/repository"
```

Reply:
Setting up resources is out of scope for individual features. The attestation section should describe how to do this rather than repeating it for every feature that relies on attestation.